go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -63,6 +61,13 @@ var (
 	// striping mode
 	xtrabackupStripes         = flag.Uint("xtrabackup_stripes", 0, "If greater than 0, use data striping across this many destination files to parallelize data transfer and decompression")
 	xtrabackupStripeBlockSize = flag.Uint("xtrabackup_stripe_block_size", 102400, "Size in bytes of each block that gets sent to a given stripe before rotating to the next stripe")
+	// switch which compressor/decompressor to use
+	builtinCompressor   = flag.String("xtrabackup_builtin_compressor", "pgzip", "which builtin compressor engine to use")
+	builtinDecompressor = flag.String("xtrabackup_builtin_decompressor", "auto", "which builtin decompressor engine to use")
+	// use and external command to decompress the backups
```

Comment:
typo: and -> an

Reply:
While adding support to `builtinbackupengine`, all these flags should be renamed so that they are not xtrabackup specific.

Reply:
hi @deepthi thanks for the feedback. I will add support for it this week in the builtin engine and for the tests 👍